examples/translation/prepare-iwslt17-multilingual.sh

```diff
@@ -78,7 +78,7 @@ for SRC in "${SRCS[@]}"; do
             | sed -e 's/<\/description>//g' \
             | sed 's/^\s*//g' \
             | sed 's/\s*$//g' \
-            > "$DATA/train.${SRC}-${TGT}.${LANG}"
+            >> "$DATA/train.${SRC}-${TGT}.${LANG}"
```

Comment:
Is this one needed? I think this should just enumerate `de-en.de`, `de-en.en`, `fr-en.fr`, `fr-en.en`, so there wouldn't be any overwriting.

Reply:
You're right, sorry! Will remove again.